gtsfm/utils/metrics.py

```diff
@@ -233,6 +233,27 @@ def compute_rotation_angle_metric(wRi_list: List[Optional[Rot3]], gt_wRi_list: L
     return GtsfmMetric("rotation_angle_error_deg", errors)
 
 
+def compute_relative_rotation_angle_metric(
+    i2Ri1_dict: Dict[Tuple[int, int], Optional[Unit3]], wRi_list: List[Optional[Pose3]]
+) -> GtsfmMetric:
+    """Computes consistency statistics between estimated global rotations and relative rotation measurements.
+
+    Args:
+        i2Ri1_dict: List of relative rotation measurements.
+        wRi_list: List of estimated camera global rotations.
+
+    Returns:
+        A GtsfmMetric for the relative rotation angle errors, in degrees.
+    """
+    angles: List[Optional[float]] = []
+    for i1, i2 in i2Ri1_dict:
+        i2Ri1 = i2Ri1_dict[(i1, i2)]
+        angles.append(
+            comp_utils.compute_rotation_angle_measurement_consistency(i2Ri1, wRi2=wRi_list[i2], wRi1=wRi_list[i1])
```

Comment:
can we not do the .between() here? do we need that extra function?

Reply:
Good point, we didn't need the extra function. I've removed it now, and consolidate the 3-4 lines of code into the caller directly.